documentation/src/main/asciidoc/topics/ref_hotrod_client_transactions.adoc

```diff
@@ -18,6 +18,12 @@ The following server configuration is required, otherwise transactions rollback
 
 * Isolation level must be `REPEATABLE_READ`.
 * `PESSIMISTIC` locking mode is recommended but `OPTIMISTIC` can be used.
++
+[NOTE]
+====
+Embedded {brandname} supports pessimistic locks but HotRod clients do not. Therefore, the transaction result obtained from using pessimistic locks in {brandname} server might differ from the result obtained from HotRod client.
```

Comment:
I would move this to line 44, inside section `== Configuring Hot Rod Clients`.

typo, hot rod is 2 words :)  `HotRod` -> `Hot Rod`

Reply:
Thanks, @pruivo, updated! WDYT?